algorithms/CPlusPlus/Maths/armstrong.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+int main() {
+int num, i=0,r,sum=0,s;
+//cout <<"enter a number to check it is Armstrong no. or not ";
+cin>>num; 
+s=num;
+while (i<num)
+{
+    r=num%10;
+    sum =sum+(r*r*r);
+    num =num/10;
+}
```

Comment:
This program is only valid for 3 digit number.
What if i give a 4 digit number or more?

Reply:
Ok, let me correct it, so that it also suits to big numbers